automapper.go

```diff
@@ -47,67 +70,89 @@ func Map(source, dest interface{}) {
 // behavior - but I'd rather not add that functionality before I have a better
 // idea what is a good options format.
 func MapLoose(source, dest interface{}) {
+	MapLooseWithOptions(source, dest, MapOptions{})
```

Comment:
Maybe I am understanding this incorrectly, but I feel like the `MapLooseWithOptions` function is not necessary. If you set the `loose` bool to `true` in the `MapLoose` function it should have the same effect as what happens on line 90 in the `MapLooseWithOptions` function
```suggestion
	MapWithOptions(source, dest, MapOptions{loose: true})
```

Reply:
You're right. I was thinking about maybe removing `MapLoose` (and `MapLooseWithOptions`) in the feature, because it might cause unsafe mappings. For now I wanted to keep the API the same as Map, but I think that might cause issues. In the current implementation `MapLooseWithOptions` with `UseSourceMemberList` will panic if a source member is not found...

I'll remove MapLooseWithOptions! :)